doc/src/modules/vector/api/classes.rst

```diff
@@ -2,13 +2,13 @@
 Essential Classes in sympy.vector (doctrings)
 =============================================
 
-CoordSysCartesian
-=================
+CoordSystem3D
```

Comment:
I'm not so favorable to using subclasses. This means that every coordinate system would require the implementation of its own class. I prefer to have a single class for all coordinate systems, the specific coordinate systems being its objects.